### ISSUE TITLE
Tabs are not navigable using the keyboard in IE8

### ISSUE THREAD

**Issue by 36degrees** [alphagov/govuk-frontend#1327]
The left / right arrow shortcuts to change between tabs are not working in IE8 (as tested using Browserstack, Windows 7).

Because we remove inactive tabs from the tab order, this means you cannot navigate between tabs using the keyboard in IE8.

**Comment by kellylee-gds** [comment]
If this isn't an easy fix we should consider falling back to the non-JS version for IE8
